assets/src/scss/components/_buttons.scss

```diff
@@ -296,7 +296,8 @@ button.load-more-mt {
   transition-duration: 100ms;
 }
 
-.wp-block-button.transparent-button a {
+.wp-block-button.transparent-button a,
```

Comment:
For now I've kept the old class to make sure existing Highlighted CTA patterns still look fine, but I'm not sure what would be best? This pattern probably hasn't been used a lot yet, so we should be able to manually update them, but I don't know if there's an easy way to find them since patterns don't appear in the blocks report... Maybe we need to run a migration script? 🤔

Reply:
I can think of ways to detect patterns, more or less reliable :grimacing: , but it would be a lot easier if we were adding a unique class to the container of each one. 
Sounds like a good idea to manually update while we can (or ping people on slack, they probably know better where their content is), by creating a ticket to do that and remove this rule after ?

Reply:
Created a [ticket](https://jira.greenpeace.org/browse/PLANET-6780)